### PR TITLE
3D data augmentation cleanup

### PR DESCRIPTION
**Pull request by lengzq**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/keras-team/keras-cv/blob/master/.github/CONTRIBUTING.md),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue? Please add a link
      to it if that's the case.
- [ ] Did you write any new necessary tests?
- [ ] If this adds a new model, can you run a few training steps on TPU in Colab to ensure that no XLA incompatible OP are used?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
 OBJECT_POINT_CLOUDS = base_augmentation_layer_3d.OBJECT_POINT_CLOUDS
 OBJECT_BOUNDING_BOXES = base_augmentation_layer_3d.OBJECT_BOUNDING_BOXES
+BOX_LABEL_INDEX = base_augmentation_layer_3d.BOX_LABEL_INDEX
```
Comment (ianstenbit) [inline keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes.py:24]: For these constant assignments -- would it make more sense to just import these symbols directly instead?

e.g. from keras_cv.layers.preprocessing3d.base_augmentation_layer_3d import BOX_LABEL_INDEX

Your call -- I don't have a strong opinion on this.

**Review by ianstenbit — APPROVED** [review]
thanks! Just one question (in-line)

File: keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes.py (hunk 1 above)
Comment (lengzq) [reply keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes.py:24]: Let's keep it this way for now and decide once all the augmentations are released?

Comment (ianstenbit) [reply keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes.py:24]: sgtm

**Review by tanzhenyu — APPROVED** [review]

**Pull request merged**